Sources/CollectionViewDriver.swift

```diff
@@ -80,14 +66,12 @@ public final class CollectionViewDriver: NSObject {
                 viewModel: CollectionViewModel = .empty,
                 options: CollectionViewDriverOptions = .init(),
                 emptyViewProvider: EmptyViewProvider?,
-                cellEventCoordinator: CellEventCoordinator?,
-                didUpdate: DidUpdate? = nil) {
+                cellEventCoordinator: CellEventCoordinator?) {
```

Comment:
Not related to this change, but I'm curious why you didn't also have the `cellEventCoordinator ` default to nil?

Reply:
That's fair. With this and `emptyViewProvider` my thinking was "you should explicitly opt-out", because otherwise you might overlook these params and then be confused about the resulting behavior.

But, now that I'm thinking about it again, I think both of these should default to `nil`. I'll push those updates separately.